app/scripts/metamask-controller.js

```diff
@@ -898,8 +898,8 @@ export default class MetamaskController extends EventEmitter {
 
         const { txReceipt } = txMeta;
 
-        // if this is a transferFrom method generated from within the app it may be a collectible transfer transaction
-        // in which case we will want to check and update ownership status of the transferred collectible.
+        // if this is a transferFrom method generated from within the app it may be a nft transfer transaction
+        // in which case we will want to check and update ownership status of the transferred nft.
```

Comment:
``` suggestion
        // if this is a transferFrom method generated from within the app it may be an NFT transfer transaction
        // in which case we will want to check and update ownership status of the transferred NFT.
```